pkgs/development/python-modules/orange-canvas-core/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, anyqt
+, cachecontrol
+, commonmark
+, dictdiffer
+, docutils
+, filelock
+, lockfile
+, pytest-qt
+, pytestCheckHook
+, python
+, qasync
+, qt5
+, writeShellScript
+, xvfb-run
+}:
+
+buildPythonPackage rec {
+  pname = "orange-canvas-core";
+  version = "0.1.31";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-kqh/c0pEWFLqf1BMD79li1MqLpH/4xrdTH9+/7YO/tg=";
+  };
+
+  propagatedBuildInputs = [
+    dictdiffer
+    commonmark
+    qasync
```

Comment:
```suggestion
```
It seems that `qasync` is not required here.
https://github.com/biolab/orange-canvas-core/blob/6523da9bb654935243da888277da3ae80883f6b0/setup.py#L24-L33

Reply:
```
> ERROR: Could not find a version that satisfies the requirement qasync (from orange-canvas-core) (from versions: none)
       > ERROR: No matching distribution found for qasync
```